src/store/enhancedCockpitApi.ts

```diff
@@ -0,0 +1,50 @@
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/redux';
+
+import { cockpitApi } from './cockpitApi';
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const errorMessage = (err: any) => {
+  let msg = err.error.statusText;
+  if (err.error.data?.errors.length > 0 && err.error.data?.errors[0]?.detail) {
+    msg = err.error.data?.errors[0]?.detail;
+  }
+  return msg;
+};
+
+const enhancedApi = cockpitApi.enhanceEndpoints({
+  addTagTypes: ['Blueprints'],
+  endpoints: {
+    getBlueprints: {
+      providesTags: () => {
+        return [{ type: 'Blueprints' }];
+      },
+    },
+    deleteBlueprint: {
```

Comment:
This is duplicate code, but it was kind of difficult to extract this because of all of the different types.